### PR TITLE
feat(api, app-shell): add 7.0.2 release notes

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

Add release notes for 7.0.2 

# Risk assessment

Low

### CODE REVIEW

File: app-shell/build/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
 
+## Opentrons App Changes in 7.0.2
+
+Welcome to the v7.0.2 release of the Opentrons App!
+
+
+### Bug Fixes
+
+- Fixes an issue where robot system updates over USB were stalling
+
```
Comment (mjhuff) [inline app-shell/build/release-notes.md:17]: Probably should add a line about fixing the issue where instruments requiring firmware updates do not display as attached on the ODD.

Comment (shlokamin) [reply app-shell/build/release-notes.md:17]: true but since there havent been any fw updates nobody should have experienced this right?

Comment (mjhuff) [reply app-shell/build/release-notes.md:17]: Oh, that's fair 👍

**Review by mjhuff — APPROVED** [review]

**Review by sfoster1 — APPROVED** [review]
Looks good to me with changes!

**Pull request merged**